src/OrbitGl/TrackContainer.h

```diff
@@ -58,6 +58,7 @@ class TrackContainer final : public CaptureViewElement {
           iterator_timer_info,
       const absl::flat_hash_map<uint64_t, uint64_t>& iterator_id_to_function_id);
   void UpdateVerticalScroll(float ratio);
+  void PanVertically(int delta);
```

Comment:
I would prefer to use "Scrolling" for vertical movement all across this PR. "Panning" is usually used to describe horizontal camera motion (originating from "Panorama" if [Wikipedia](https://en.wikipedia.org/wiki/Panning_(camera)) is to be believed). We're also using "scroll" basically everywhere else when we refer to vertical movement (see the method right above this one).

Reply:
Thanks!